pipelines/katello_with_pulp3.yml

```diff
@@ -0,0 +1,26 @@
+---
+- hosts: localhost
+  vars:
+    forklift_name: katello-with-pulp3
+    forklift_boxes:
+      katello-with-pulp3:
+        box: centos7
+        memory: 10240
+  roles:
+    - forklift
+
+- hosts: katello-with-pulp3
+  become: true
+  roles:
+    - role: forklift_versions
+      scenario: "katello"
+      scenario_version: "3.12"
```

Comment:
Any reason this isn't using nightly? Perhaps at least update it to 3.13 now that it's GA.